CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 22.02+ (???)
 ------------------------------------------------------------------------
-- Feature: [#1327] Readd the game intro (use commandline switch --intro to enable)
+- Feature: [#1327] Readd the game intro (use commandline switch --intro to enable).
+- Feature: [#1347] Add OpenAL-soft audio engine.
```

Comment:
Perhaps we should list it as 'Technical', as it's not user-facing or optional. We should probably also mention that it replaces SDL2_mixer?

Reply:
I'll leave the SDL2_mixer part till later when we actually drop the dependency.